design-documents/asynchronous-import/base-extension.md

```diff
@@ -279,30 +256,79 @@ Will be returned list of objects that we tried to import
 ```
 {
   "status": "string", 
-  "error": "string",
-  "uuid": 0,
-  "entity_type": "catalog_product, customers ....",
-  "user_id": "User ID who created this request",
-  "user_type": "User Type who created this request",
-  "items": [
-  {
-    "uuid": 0,
-    "status": "",
-    "serialized_data": "",
-    "result_serialized_data": "",
-    "error_code":"",
-    "result_message":""
-  }]
+  "errors": [],
+  "created_at": "string",
+  "finished_at": "string",
+  "import": {
+        "uuid": "",
+        "source_uuid": "",
+        "import_type": "catalog_product",
+        "validation_strategy": "string",
+        "allowed_error_count": 0,
+        "import_image_archive": "string",
```

Comment:
are we planning to import images as well? if yes, not sure where they will be landing, it could be some CDN etc.

Reply:
Yes we are planning and currently they will be imported in Magento.
Magento our of box is saving all images under pub/media/
Magento API same time accepting images as base64_encoded content.
So form our side are not really care about where are the images saved, we just taking what customer giving to us, and forwarding them to Magento in base64 view. 

Customer providing images by uploading them on Magento filesystem. and they will be deleted after import.

Yes, in perfect way we have to support CDN storage, and for Magento we already have some support. But for our module - its currenly I guess out of scope of MVP.

Interesting point for future, we can groom locating where User can user place images for Import